extensions/ql-vscode/src/skeleton-query.ts

```diff
@@ -80,6 +82,18 @@ export class SkeletonQueryWizard {
     });
   }
 
+  private workoutStoragePath() {
```

Comment:
`workoutStoragePath` seems quite general. Would `getFirstStoragePath` work better as a name that describes what this is doing?